### DISCUSSION TITLE
Tags and Backlinks

### DISCUSSION THREAD

**Discussion by aniket-rgb** [TriliumNext/Trilium#3361] [Q&A] [answered]
I want to use [zettelkasten method](https://zettelkasten.de/posts/overview/#the-introduction-to-the-zettelkasten-method)

1. How to tag notes?
2. Is there any way to backlink in trilium?
e.g. in apps like zettlr or athens, typing `[[`, shows the other notes to backlink.

**Comment by fooyoo** [comment, marked as answer]
1 https://github.com/zadam/trilium/wiki/Attributes
2 '@' can be equivalent to [[, see https://github.com/zadam/trilium/wiki/Links
backlink will be the upper right corner of the note.
![backlink](https://user-images.githubusercontent.com/36770100/204781907-8461af91-8d28-4daf-abae-379117bc8d81.png)
